sdk/core/azure-core/src/main/java/com/azure/android/core/util/ServiceVersion.java

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.android.core.util;
+
+/**
+ * A generic interface for sending HTTP requests using the provided service version.
```

Comment:
Maybe we don't have to tie it to HTTP? something like "A generic interface for sending service requests using the provided service version." thoughts?

Reply:
Good catch, I'll make the change